mobile_config/src/client/mod.rs

```diff
@@ -21,22 +24,39 @@ pub enum ClientError {
     VerificationError(#[from] file_store::Error),
 }
 
-#[derive(Clone, Debug)]
+#[derive(Clone)]
 pub struct Client {
     pub client: mobile_config::GatewayClient<Channel>,
     signing_key: Arc<Keypair>,
     config_pubkey: PublicKey,
     batch_size: u32,
+    cache: Arc<Cache<PublicKeyBinary, Option<gateway_info::GatewayInfo>>>,
```

Comment:
the cache doesn't implement Debug i take it...

Reply:
It does not